website/addons/base/__init__.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, short_name, full_name, owners, categories,
                  node_settings_model=None, user_settings_model=None, include_js=None, include_css=None,
                  widget_help=None, views=None, configs=None, models=None,
                  has_hgrid_files=False, get_hgrid_data=None, max_file_size=None, high_max_file_size=None,
-                 accept_extensions=True,
+                 accept_extensions=True, description='', url=None,
```

Comment:
I don't think this PR is the straw that broke the camel's back... but whenever I see a constructor with this many arguments, I start thinking about ways to refactor and encourage separation of concerns. 

As an actionable example, user-facing text seems like it would be a property of the addon itself, not the configuration object. I count 20+ arguments (plus kwargs) here, which is something to consider going forward.